package.json

```diff
@@ -1410,6 +1410,7 @@
     "@kbn/whereis-pkg-cli": "link:packages/kbn-whereis-pkg-cli",
     "@kbn/yarn-lock-validator": "link:packages/kbn-yarn-lock-validator",
     "@mapbox/vector-tile": "1.3.1",
+    "@mswjs/http-middleware": "^0.10.1",
```

Comment:
It looks like we didn't have `@mswjs/http-middleware` in 8.15, do we need to add it?

Reply:
Removed in d91c35bc31b272bdd151d7125170e2ebfba59792